project_national/national_upgrades_HPs.yml

```diff
@@ -0,0 +1,162 @@
+project_directory: project_national # Relative to buildstock_directory
+output_directory: national_upgrades_HPs
+weather_files_url: https://data.nrel.gov/system/files/156/Buildstock_TMY3_FIPS-1678817889.zip
+# weather_files_path: c:/OpenStudio/BuildStock_TMY3_FIPS.zip
+
+sampler:
+  type: residential_quota
+  args:
+    n_datapoints: 5
+
+workflow_generator:
+  type: residential_hpxml
+  args:
+    build_existing_model:
+      simulation_control_timestep: 60
+      simulation_control_run_period_begin_month: 1
+      simulation_control_run_period_begin_day_of_month: 1
+      simulation_control_run_period_end_month: 12
+      simulation_control_run_period_end_day_of_month: 31
+      simulation_control_run_period_calendar_year: 2007
+
+    emissions:
+      - scenario_name: LRMER_MidCase_15
+        type: CO2e
+        elec_folder: data/emissions/cambium/2022/LRMER_MidCase_15
+
+    utility_bills:
+      - scenario_name: Bills
+
+    simulation_output_report:
+      timeseries_frequency: hourly
+      include_timeseries_total_consumptions: true
+      include_timeseries_fuel_consumptions: true
+      include_timeseries_end_use_consumptions: true
+      include_timeseries_emissions: true
+      include_timeseries_emission_fuels: true
+      include_timeseries_emission_end_uses: true
+      include_timeseries_hot_water_uses: true
+      include_timeseries_total_loads: true
+      include_timeseries_component_loads: true
+      include_timeseries_unmet_hours: true
+      include_timeseries_zone_temperatures: true
+      include_timeseries_airflows: true
+      include_timeseries_weather: true
+      include_timeseries_resilience: true
+
+    reporting_measures:
+      - measure_dir_name: QOIReport
+
+    server_directory_cleanup:
+      retain_in_idf: false
+      retain_schedules_csv: false
+
+baseline:
+  n_buildings_represented: 139647020 # American Community Survey 2021 5-year, B25001, does not include territories ( 138765649 without AK and HI )
+
+upgrades:
+  # ASHP base scenario
+  - upgrade_name: ASHP Base
+    package_apply_logic:
+      - HVAC Has Ducts|Yes
+    options:
+      - &hvac_heating_efficiency_ashp_seer_22_10_hspf
+        option: HVAC Heating Efficiency|ASHP, SEER 22, 10 HSPF
+        costs:
+          - value: 50.0
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: 30
+      - &hvac_cooling_efficiency_ducted_heat_pump
+        option: HVAC Cooling Efficiency|Ducted Heat Pump
+      - &heat_pump_backup_use_existing_system
+        option: Heat Pump Backup|Use Existing System
+
+  # ASHP w/ ductwork restriction scenario
+  - upgrade_name: ASHP Ductwork Restriction
+    package_apply_logic:
+      - HVAC Has Ducts|Yes
+    options:
+      - *hvac_heating_efficiency_ashp_seer_22_10_hspf
+      - *hvac_cooling_efficiency_ducted_heat_pump
+      - *heat_pump_backup_use_existing_system
+      - &hvac_distribution_ductwork_restriction
+        option: HVAC Distribution|Ductwork Restriction
```

Comment:
Test number of datapoints somewhere on the order of hundreds? Sizing at MaxLoad.